<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSettings</class>
 <widget class="QDialog" name="dlgSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPChecker Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/IPChecker.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="layoutTitle">
     <item>
      <widget class="QLabel" name="lblIcon">
       <property name="font">
        <font>
         <family>AlArabiya</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/IPChecker.qrc">:/icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="font">
        <font>
         <family>AlArabiya</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IPChecker Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblVersion">
       <property name="text">
        <string>v1.2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="layoutServer">
     <item>
      <widget class="QLabel" name="lblServer">
       <property name="text">
        <string>&amp;Server:</string>
       </property>
       <property name="buddy">
        <cstring>txtServer</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtServer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="layoutPort">
     <item>
      <widget class="QLabel" name="lblPort">
       <property name="text">
        <string>&amp;Port:</string>
       </property>
       <property name="buddy">
        <cstring>spinPort</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinPort">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>21</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="layoutDir">
     <item>
      <widget class="QLabel" name="lblDir">
       <property name="text">
        <string>&amp;Dir:</string>
       </property>
       <property name="buddy">
        <cstring>txtDir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layoutUsername">
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="text">
        <string>&amp;Username:</string>
       </property>
       <property name="buddy">
        <cstring>txtUsername</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtUsername"/>
     </item>
    </layout>
   </item>
   <item row="2" column="2" colspan="2">
    <layout class="QHBoxLayout" name="layoutPassword">
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Pass&amp;word:</string>
       </property>
       <property name="buddy">
        <cstring>txtPassword</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="layoutPrimary">
     <item>
      <widget class="QLabel" name="lblPrimary">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Primary &amp;IP Provider:</string>
       </property>
       <property name="buddy">
        <cstring>txtPrimaryIPProvider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPrimaryIPProvider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="layoutBackup">
     <item>
      <widget class="QLabel" name="lblBackup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Backup IP Provider:</string>
       </property>
       <property name="buddy">
        <cstring>txtBackupIPProvider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtBackupIPProvider">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnAutoSetup">
       <property name="text">
        <string>&amp;IPXHost Auto-Setup</string>
       </property>
       <property name="icon">
        <iconset resource="../res/IPChecker.qrc">
         <normaloff>:/restore.png</normaloff>:/restore.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResetIPChecker">
       <property name="text">
        <string>&amp;Reset IP Checker</string>
       </property>
       <property name="icon">
        <iconset resource="../res/IPChecker.qrc">
         <normaloff>:/reset.png</normaloff>:/reset.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QHBoxLayout" name="layoutBottom">
     <item>
      <widget class="QDialogButtonBox" name="btnboxDialog">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnboxDialog</tabstop>
  <tabstop>txtServer</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>txtPrimaryIPProvider</tabstop>
  <tabstop>btnAutoSetup</tabstop>
 </tabstops>
 <resources>
  <include location="../res/IPChecker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnboxDialog</sender>
   <signal>accepted()</signal>
   <receiver>dlgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnboxDialog</sender>
   <signal>rejected()</signal>
   <receiver>dlgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
